assets/queries/k8s/limit_capabilities_of_the_psp/metadata.json

```diff
@@ -1,8 +1,8 @@
 {
   "id": "limit_the_capabilities_from_the_psp",
-  "queryName": "Limit The Capabilities From The PSP",
+  "queryName": "Not Limited Capabilities For Pod Security Policy",
```

Comment:
Unlimited?

Reply:
Prefer the use of Not Limited in this case